web/src/routes/OrganizationsList/Organization/Tabs/Settings/BillingInformation.tsx

```diff
@@ -0,0 +1,454 @@
+        return [
+          ...prevAlerts,
+          <Alert
+            key="alert"
+            variant="danger"
+            title={err.response.data.error_message}
+            isInline={true}
+            timeout={5000}
+          />,
+        ];
+      });
+    },
+  });
+
+  const [accountType, setAccountType] = useState(
+    isUserOrganization ? 'individual' : 'organization',
+  );
+  useEffect(() => {
+    setAccountType(isUserOrganization ? 'individual' : 'organization');
+  }, [loading]);
+  const {currentPlan, privateAllowed, privateCount, upgrade} = useUpgradePlan(
+    organizationName,
+    accountType == 'organization',
+  );
+
+  const {
+    convert,
+    loading: convertAccountLoading,
+    error: convertAccountError,
+  } = useConvertAccount({
+    onSuccess: (result) => {
+      setAlerts((prevAlerts) => {
+        return [
+          ...prevAlerts,
+          <Alert
+            key="alert"
+            variant="success"
+            title="Successfully updated settings"
+            isInline={true}
+            timeout={5000}
+          />,
+        ];
+      });
+    },
+    onError: (err) => {
+      setAlerts((prevAlerts) => {
+        return [
+          ...prevAlerts,
+          <Alert
+            key="alert"
+            variant="danger"
+            title={err.response.data.error_message}
+            isInline={true}
+            timeout={5000}
+          />,
+        ];
+      });
+    },
+  });
+
+  const error = userUpdateError || convertAccountError;
+
+  const updateLoading = userUpdateLoading;
+  useEffect(() => {
+    resetFields();
+  }, [loading]);
+
+  const resetFields = () => {
+    if (!loading && organization) {
+      setInvoiceEmail(organization.invoice_email || false);
+      setInvoiceEmailAddress(organization.invoice_email_address || '');
+      if (organization.invoice_email_address) {
+        setValidated('success');
+      } else {
+        setValidated('default');
+      }
+    } else if (isUserOrganization) {
+      setInvoiceEmail(user.invoice_email || false);
+      setInvoiceEmailAddress(user.invoice_email_address || '');
+      if (user.invoice_email_address) {
+        setValidated('success');
+      } else {
+        setValidated('default');
+      }
+    }
+    setTouched(false);
+  };
+
+  const {totalResults} = useRepositories(organizationName);
+
+  return (
+    <Form id="form-form" width="70%">
+      {error && (
+        <FormAlert>
+          <Alert
+            variant="danger"
+            title={((error as AxiosError).response.data as any).error_message}
+            aria-live="polite"
+            isInline
+          />
+        </FormAlert>
+      )}
+      <Flex
+        spaceItems={{default: 'spaceItemsSm'}}
+        direction={{default: 'column'}}
+      >
+        <FlexItem>
+          <Title headingLevel="h3">
+            {currentPlan?.plan.toUpperCase()} organization&apos;s plan
+          </Title>
+        </FlexItem>
+        {privateAllowed ? (
```

Comment:
nit: there's the `Conditional` component that may make this easier to read. More of a matter of preference. 
```
<Conditional if={privateAllowed}>
    <FlexItem>{`${privateCount} of ${privateAllowed} private repositories used`}</FlexItem>
</Conditional>
```